src/csvcubed/utils/uri.py

```diff
@@ -112,3 +112,21 @@ def ensure_values_in_lists_looks_like_uris(values: list[str]) -> None:
             raise ValueError(f"'{value}' does not look like a URI.")
 
     _logger.debug("Values %s all look like URIs.", values)
+
+
+def extract_uri_template_variable_name_by_index(uri_template: str, index: int = 0) -> str:
+    """
+    Extract variable name from a uri template by index.
+
+    Usage:
+    
+    index 0: "http://example.com/something/{+foo}" = "foo" 
+    
+    index 0: "http://example.com/something/{bar}" = "bar"
+    
+    index 0: "http://example.com/{one}/segment/{two}#stuff = "one"
+    
+    index 1: "http://example.com/{one}/segment/{two}#stuff = "two"
+    """
+    vars = [x.split("{")[1] for x in uri_template.split("}")[:-1]]
```

Comment:
We're already using the [uritemplate](https://uritemplate.readthedocs.io/en/latest/) package for [stuff like this](https://github.com/GSS-Cogs/csvcubed/blob/55905b98a50c452f0ed999cc8f7ea6f84eae3e22/src/csvcubed/readers/skoscodelistreader.py#L54).